component/responsive image/image.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">

```

Comment:
Just a suggestion, you can use custom meta info just to give a personal touch. For eg: `<meta name="author" content="Sneha Singh">`

Reply:
great thanks